sw/device/lib/arch/device_fpga_nexysvideo.c

```diff
@@ -11,13 +11,13 @@
 
 const device_type_t kDeviceType = kDeviceFpgaNexysVideo;
 
-const uint64_t kClockFreqCpuHz = 50 * 1000 * 1000;  // 50MHz
+const uint64_t kClockFreqCpuHz = 10 * 1000 * 1000;  // 10MHz
 
-const uint64_t kClockFreqPeripheralHz = 125 * 100 * 1000;  // 12.5MHz
+const uint64_t kClockFreqPeripheralHz = 25 * 100 * 1000;  // 2.5MHz
 
 const uint64_t kClockFreqUsbHz = 48 * 1000 * 1000;  // 48MHz
 
-const uint64_t kUartBaudrate = 230400;
+const uint64_t kUartBaudrate = 57600;
```

Comment:
Do we really need to do that? Off-chip transfer is already rather slow (especially when having a fair number of printf() messages in tested software). I think our UART uses 16x oversampling, so 115200 should be possible. (According to the error calculation in the UART OT doc even 230400, but I'm not sure that's actually correct.)

Reply:
If we make any change to this frequency, we need to also change it in the docs.

Reply:
yeah agreed with Phil, I don't think we have to slow down the uart baud rate.
The software is also setup to recalculate based on target baudrate.  So I think it should be okay.

Reply:
@eunchan with 230400 baud and 2.5 MHz frequency we can't do 16x oversampling any more. Is that OK?

Reply:
o wait... you're right.. i forgot it was 2.5MHz, i  was thinking 10MHz in my head..
Eunchan and I went through an exercise on this recently, and you definitely need the 16, I think lower than 16 the software might even error out on the programming.

Reply:
So that means we either need to go with 115200 baud or choose a higher peripheral clock. Probably 115200 baud is the easier option.

Reply:
Yep, I have switched to 115200 and also adjusted the docs.